cdcsdk-server/cdcsdk-server-kafka/src/main/java/io/debezium/server/kafka/KafkaChangeConsumer.java

```diff
@@ -98,8 +98,7 @@ public void handleBatch(final List<ChangeEvent<Object, Object>> records, final R
 
                 if (key != null) {
                     this.metrics.get(Metrics.bytesWritten).increment(
-                            key.getBytes().length
-                    );
+                            key.getBytes().length);
```

Comment:
This is a change the auto-formatter is doing every time. Will remove this and will try to address this in a separate PR wherein I'll modify the auto-formatter properties as well.